flatdata-generator/flatdata/generator/templates/rust/rust.jinja2

```diff
@@ -8,8 +8,16 @@
 {%- for ns in tree.root.children recursive %}
 {%- if ns | is_namespace %}
 
+{% if ns | is_builtin_namespace %}
+#[doc(hidden)]
```

Comment:
Nice...

I have also been thinking: Most likely we should move the schemas to the bottom of the file... they not the least useful thing to rread

Reply:
Makes sense. I will do in the next PR, since otherwise the diff will become crazy. https://github.com/heremaps/flatdata/issues/168